templates/server/index.html

```diff
@@ -38,81 +32,39 @@ <h1>Scale out with Ubuntu Server</h1>
 
 <div class="p-strip is-bordered">
   <div class="u-fixed-width">
-    <h2>
-      What's new in {{ releases.latest.full_version }}
+    <h2> What's new in {{ releases.lts.short_version }}
     </h2>
     <ul class="p-list is-split">
-      <li class="p-list__item is-ticked">Supported by Canonical until {{ releases.latest.eol }}</li>
-      <li class="p-list__item is-ticked">Certified native drivers for NVIDIA virtual GPU (vGPU) software on Ubuntu 20.04 LTS and Ubuntu 18.04 LTS <sup>*</sup></li>
-      <li class="p-list__item is-ticked">Minimal system installation option through the Ubuntu Server Live Installer</li>
-      <li class="p-list__item is-ticked"><a href="http://manpages.ubuntu.com/manpages/xenial/man1/needrestart.1.html">Needrestart</a> by default for automated daemon restarts after applying library upgrades</li>
-      <li class="p-list__item is-ticked">The latest stable Linux 5.13 kernel for the latest hardware and security updates</li>
-      <li class="p-list__item is-ticked">Runs on all major architectures - x86-64, ARM v7, ARM64, POWER8, POWER9, IBM s390x (LinuxONE) and RISC-V</li>
-      <li class="p-list__item is-ticked">Updates to QEMU (6.0), libvirt (7.6), PHP (8.0.8), Apache2 (2.4.48), GCC (11.2.0), Python (3.9.4), Bind9 (9.16.15), Open vSwitch (2.16.0) and OpenLDAP (2.5.6)</li>
-    </ul>
-    <p>
-      <small>
-        <sup>*</sup> available to <a href="/advantage">Ubuntu Advantage</a> customers only
-      </small>
-    </p>
-    <p>
-      <a
-     
-      href="{{ releases.latest.release_notes_url }}">
-      Read more in the release notes
-    </a>
-  </p>
-</div>
-</div>
-
-<div class="p-strip is-bordered">
+      <li class="p-list__item is-ticked"><a href="/support">Supported by Canonical</a> for 10 years until April 2027</li>
```

Comment:
The copy doc shows this as being supported until 2032 - shouldn't that be in the releases.yaml under `{{ releases.latest.eol }}`?